### PR TITLE
fix(gemini): unthemed elements

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image
image

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/google-gemini/catppuccin.user.css [hunk 1]
```diff
@@ -178,7 +178,12 @@
     --bard-color-tunable-selection-textarea-background: @base;
     --bard-color-tunable-selection-shimmer-non-advanced: @accent-color;
     --bard-color-action-tooltip-background: darken(@accent-color, 20%);
-
+    --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--on-surface: @text;
+    --gem-sys-color--outline-variant: @text;
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:183]:
```suggestion
    --gem-sys-color--outline-variant: @subtext0;
```

**Review by Dandraghas — CHANGES_REQUESTED** [review]

**Review by Dandraghas — CHANGES_REQUESTED** [review]
A lot of text is still unstyled, switch to latte in stylus and you will see

**Review by Dandraghas — COMMENTED** [review]
chat list
image

disclaimer
image

code blocks
image
image

extensions card description
image

location footer text doesnt follow accent-color(--gem-sys-color--primary)
image

File: styles/google-gemini/catppuccin.user.css [hunk 2]
```diff
@@ -232,9 +239,19 @@
 
     --og-theme-color: @text;
 
+    span {
+        color: @text
+    }
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:244]: What is this for? if you are trying to style "From your ip address" text you should use `--gem-sys-color--primary: @accent-color;

also you missing semicolon

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css:244]: this is for the other text that can't be themed using vars

**Review by Dandraghas — CHANGES_REQUESTED** [review]
huh, what happened to reviews

File: styles/google-gemini/catppuccin.user.css [hunk 3]
```diff
@@ -232,9 +239,19 @@
...
+        color: @text;
+    }
+    
     .button-pressed {
       --bard-color-primary: @mantle;
     }
+    
+    .textarea, .text-input-field {
+        background-color: @mantle;
+        color: @text
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:252]:
```suggestion
        color: @text;
```

File: styles/google-gemini/catppuccin.user.css [hunk 4]
```diff
@@ -178,7 +178,14 @@
...
     --bard-color-action-tooltip-background: darken(@accent-color, 20%);
-
+    --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--on-surface: @text;
+    --gem-sys-color--outline-variant: @subtext0;
+    --gem-sys-color--surface-container-low: @mantle;
+    --gem-sys-color--outline: @accent-color;
+    --gem-sys-color--on-surface-variant: @text;
+    --gem-sys-color--on-primary-container: @text;
+
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:188]:
```suggestion
    --gem-sys-color--on-primary-container: @text;
    --gem-sys-color--primary: @accent-color;
    
```

**Review by Dandraghas — COMMENTED** [review]
https://gemini.google.com/advanced is unstyled

innactive tab color can be styled using --mat-tab-header-inactive-label-text-color
image

card container(--bard-color-adv-marketing-basic-background)  and checkmark(--bard-color-adv-marketing-check) is incorrect color
image

**Comment by somerand0mcat** [comment]
i'll fix it dw

File: styles/google-gemini/catppuccin.user.css [hunk 5]
```diff
@@ -178,7 +178,17 @@
...
+    --bard-color-adv-marketing-check: @text;
+    --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--on-surface: @text;
+    --gem-sys-color--outline-variant: @subtext0;
+    --gem-sys-color--surface-container-low: @mantle;
+    --gem-sys-color--outline: @accent-color;
+    --gem-sys-color--on-surface-variant: @text;
+    --gem-sys-color--on-primary-container: @text;
+    --gem-sys-color--primary: @accent-color;
+    --mat-tab-header-inactive-label-text-color: @text;
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:190]:
```suggestion
    .mat-mdc-tab-group, .mat-mdc-tab-nav-bar {
      --mat-tab-header-inactive-label-text-color: @subtext0;
      --mat-tab-header-inactive-hover-label-text-color: @subtext0;
      --mat-tab-header-inactive-focus-label-text-color: @subtext0;
    }
```

File: styles/google-gemini/catppuccin.user.css [hunk 6]
```diff
@@ -232,9 +246,24 @@
...
     --og-theme-color: @text;
 
+    span {
+        color: @text;
+    }
+
+    .new-feature {
+        background: @accent-color;
+        color: @base;
+    }
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:256]: Why is it @accent-color. surface or overlay will be more appropriate here

```suggestion
    .new-feature {
        background: @surface2;
    }
```

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css:256]: i thought ``@accent-color`` was appropriate for this

File: styles/google-gemini/catppuccin.user.css (hunk 2 above)
Comment (Dandraghas) [reply styles/google-gemini/catppuccin.user.css]: This overrides some of variables

can you tell me which text cant be styled? i cant find any

**Review by Dandraghas — CHANGES_REQUESTED** [review]
One last thing

this button is unstyled
image

```
   .mat-mdc-fab.mat-accent {
      --mat-fab-foreground-color: @text;
    }
```

also card is still wrong color
change --bard-color-adv-marketing-basic-background to surface0

File: styles/google-gemini/catppuccin.user.css (hunk 2 above)
Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css]: "New chat" button and the location at the bottom

File: styles/google-gemini/catppuccin.user.css [hunk 7]
```diff
@@ -178,7 +178,21 @@
     --bard-color-tunable-selection-textarea-background: @base;
     --bard-color-tunable-selection-shimmer-non-advanced: @accent-color;
     --bard-color-action-tooltip-background: darken(@accent-color, 20%);
-
+    --bard-color-adv-marketing-check: @text;
+    --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--on-surface: @text;
+    --gem-sys-color--outline-variant: @subtext0;
+    --gem-sys-color--surface-container-low: @mantle;
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:185]: mini code block
image

```suggestion
    --gem-sys-color--outline-variant: @subtext0;
    --gem-sys-color--surface-container-high: @surface0;
    --gem-sys-color--surface-container-low: @mantle;
```

File: styles/google-gemini/catppuccin.user.css (hunk 2 above)
Comment (Dandraghas) [reply styles/google-gemini/catppuccin.user.css]: New chat button, can be styled using
```
--bard-color-on-surface-rgb: #rgbify(@text) [];
--gem-sys-color--on-surface--rgb: #rgbify(@text) [];
--gem-sys-color--surface-dim--rgb: #rgbify(@crust) [];
```

Location seems to be styled, not sure what you meant

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css]: > New chat button can be styled using
> 
> ```
> --bard-color-on-surface-rgb: #rgbify(@text) [];
> --gem-sys-color--on-surface--rgb: #rgbify(@text) [];
> --gem-sys-color--surface-dim--rgb: #rgbify(@crust) [];
> ```
> 
> Location seems to be styled, not sure what you meant

i guess i'll remove span and add these variables

File: styles/google-gemini/catppuccin.user.css [hunk 8]
```diff
@@ -232,9 +254,19 @@
...
+    
     .button-pressed {
       --bard-color-primary: @mantle;
     }
+    
+    .textarea, .text-input-field {
+        background-color: @mantle;
+        color: @text;
+    }
+
```
Comment (Dandraghas) [inline styles/google-gemini/catppuccin.user.css:269]: What the point of this? input field is already styled using --gem-sys-color--surface-container

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css:269]: that's so when you click on the text box it doesn't change color

Comment (Dandraghas) [reply styles/google-gemini/catppuccin.user.css:269]: Why, does it need to do that? in original gemini without catppuccin, its default behaviour to change colors to more lighter.

i see no reason for it to not do that in catppuccin.

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css:269]: i thought it didn't do that

Comment (somerand0mcat) [reply styles/google-gemini/catppuccin.user.css]: anyways im going to change that

**Review by Dandraghas — APPROVED** [review]

**Review by Dandraghas — APPROVED** [review]
Sorry for a wait, i forgot about this

**Pull request merged**